### ISSUE TITLE
[Feature Request] Directory of pdf files

### ISSUE THREAD

**Issue by Baidya99** [Xonshiz/comic-dl#34]
Could you please place the pdf files (or .cbz or .epub) in the root directory of the comic and not in the Issue folder.

Eg: 
XYZ 2016 Issue-1.pdf
should be in
XYZ 2016/
and not in
XYZ 2016/Issue-1/

Looks better that way.
Thanks in advance.

**Comment by Xonshiz** [comment]
No, the structure will probably remain same as to avoid any file name conflict. Readcomiconline.to isn't the only website this script supports. Manga and other comic website don't follow the "Issue" thing.

**Comment by Baidya99** [comment]
Atleast give an option.....

No problem. Great work by the way.

**Comment by Xonshiz** [comment]
In all honesty, I don't really want to write an extra module for just one website. I MIGHT come up with some other file naming pattern that will not cause conflicts.

**Comment by Baidya99** [comment]
@Xonshiz I have made a GUI for the binary.
I have attached the file.
You just have to extract the file and place it in the folder containing comic_dl.exe


[GUI.zip](https://github.com/Xonshiz/comic-dl/files/1139483/GUI.zip)

**Comment by Xonshiz** [comment]
It's great that you made a GUI, but I don't want to run an .exe from untrusted source on my system atm. I'll look into it later on.

**Comment by Baidya99** [comment]
Okay

**Comment by Baidya99** [comment]
By the way how to prove it doesn't have any virus or malware.

**Comment by Xonshiz** [comment]
why don't you make a repo for it and share that? That'd be better.

**Comment by Baidya99** [comment]
Okay

**Comment by Baidya99** [comment]
Here is the [repo](https://github.com/Baidya99/Comic-DL-GUI)

Download link in Wiki.

**Comment by Xonshiz** [comment]
Hey, just tested it! It's amazing! It could do some changes with the graphics. Hmmm... I was working on a GUI for Comic_dl as well.. in C#. There seems to be an issue with your version. When I selected a "download location", it froze the UI and mentioned that it had an issue. But, it downloaded just fine without giving any download location.

Amazing work!

**Comment by Xonshiz** [comment]
There's a problem with your GUI. It freezes the UI, while downloading. Maybe you should make an async method and put an await to to... you know, wait for the thread to complete its work without freezing the UI.

I'm a little busy with some work this month, I'll try to finish up the UI I made by the end of this month.

**Comment by Baidya99** [comment]
No I have disabled the GUI when It downloads so that you would not start while the program is running.
Only after downloading can you give the arguments again.

**Comment by Baidya99** [comment]
Let me look into the the download location problem.

**Comment by Xonshiz** [comment]
Ohhh, I see. Nice, I like it. The UI I had in mind is a little bit different. I have a test broken project lying around somewhere on my system... I'll have to re-build it and fix things accordingly.

**Comment by Xonshiz** [comment]
Just a rule of thumb, when you post your project/share it, make sure you clean the build.

Build --> Clean Solution

**Comment by jaxxa** [comment]
Or even better setup a .gitignore file.

I have not looked at it yet as I am not at home but a GUI sounds nice.
I made a small program for personal use that renames the images and packaging into .cbz files.
I can look into integrating that into your GUI if you like.

https://github.com/jaxxa/ComicCreator

**Comment by Baidya99** [comment]
Okay
But I have a problem.
My exam will start and I will be unable to code during that period.
But as soon as they get over I will work on it.

**Comment by Baidya99** [comment]
What is wrong with this command;
`comic_dl -dd "C:\Users\Baidyanath Kundu\Documents\Dead\" -i "http://readcomiconline.to/Comic/Deadpool-2016"`

I am getting error:

> I need an Input URL to download from.

**Comment by Xonshiz** [comment]
command looks fine.. try with verbose

**Comment by Baidya99** [comment]
DEBUG: Arguments Provided : Namespace(convert='None', download_directory=['C:\\Users\\Baidyanath Kundu\\Documents\\Dead" -i http://readcomiconline.to/Comic/Deadpool-2016'], input=None, keep='True', range='All', sorting=None, verbose=True, version=False)
DEBUG: Operating System : Windows - 10 - 10.0.15063
DEBUG: Python Version : 2.7.13 (32bit)

**Comment by Xonshiz** [comment]
Try this : 

`comic_dl -i "http://readcomiconline.to/Comic/Deadpool-2016" -dd "C:\Users\Baidyanath Kundu\Documents\Dead\"`

**Comment by Xonshiz** [comment]
Oh wait, seems like the arguments are being parsed in a wrong way.
`download_directory=['C:\Users\Baidyanath Kundu\Documents\Dead" -i http://readcomiconline.to/Comic/Deadpool-2016'`

`-i` is being taken into `-dd`

**Comment by Xonshiz** [comment]
I checked it on my end as well.This is a BUG. Could you please open a new issue with this?

**Comment by Xonshiz** [comment]
This should be fixed now.
